src/Modules/Station/Components/OnlineUsers/OnlineUsers.tsx

```diff
@@ -85,50 +49,121 @@ export class OnlineUsersComponent extends BaseComponent<IProps, IState> {
     return userInfo !== null;
   };
 
-  public isMe = (username?: string) => {
-    // TODO: enable after fix fetch user info
-    // if (username && this.props.userInfo) {
-    //   return this.props.userInfo.username === username;
-    // }
-    if (username && currentUser) {
-      return currentUser.username === username;
+  // TODO: using username instead id when backend support it
+  // check isCurrentUser by id
+  public isCurrentUser = (id: string) => {
+    const { userInfo: currentUser } = this.props;
+    if (id && currentUser) {
+      return currentUser.id === id;
     }
     return false;
   };
 
-  public renderPopoverContent() {
-    const filteredUsers = fixture.sort(
-      user => (user.username === currentUser.username ? 1 : 0),
+  public renderPopoverContent(onlineUsers: any[], numberOnline: 0) {
+    const { userInfo: currentUser } = this.props;
+
+    const filteredUsers = onlineUsers.sort(
+      user => (user.username === currentUser.username ? 0 : 1),
     );
+
+    // Only display keep top 10 users
```

Comment:
LIST_USERS_SHOW = 8. Could you update the comment to be appropriate or remove it